packages/yourloops/pages/hcp/teams-page.tsx

```diff
@@ -160,28 +160,24 @@ function TeamsPage(): JSX.Element | null {
     return false;
   };
 
-  const handleShowAddMemberDialog = async (team: Team): Promise<void> => {
-    log.debug("handleShowAddMemberDialog:", { team });
-
-    const getMemberEmail = () =>
-      new Promise((resolve: (result: { email: string | null; role: Exclude<TypeTeamMemberRole, "patient"> }) => void): void => {
-        setAddMember({ team, onDialogResult: resolve });
-      });
-
-    const { email, role } = await getMemberEmail();
-    setAddMember(null);
-    if (email === null) {
-      return;
+  const onMemberInvited = async (member: { email: string; role: Exclude<TypeTeamMemberRole, "patient">, team: Team } | null) => {
+    if (member) {
+      await teamHook.inviteMember(member.team, member.email, member.role);
```

Comment:
Duplicate call (in the try catch below) ? 
```suggestion
```

Reply:
good catch :sweat_smile: